static/app/views/starfish/utils/getModuleFromLocation.tsx

```diff
@@ -0,0 +1,19 @@
+import {Location} from 'history';
+
+import {t} from 'sentry/locale';
+
+export const MODULES = {
+  api: t('API Module'),
+  database: t('DB Module'),
+  'endpoint-overview': t('Endpoint Overview'),
+};
```

Comment:
This makes sense, but since the `endpoint overview` route is here as well, I think you can lift this out and into its own file, and add the other page names too. Maybe it wants to be called something like `ROUTE_NAMES`?